operator/v1alpha1/types_image_content_source_policy.go

```diff
@@ -65,3 +69,25 @@ type RepositoryDigestMirrors struct {
 	// +optional
 	Mirrors []string `json:"mirrors"`
 }
+
+// RepositoryMirrors holds cluster-wide information about how to handle mirrors in the registries config.
+// Note: this is different from the RepositoryDigestMirrors that the mirrors only
+// work when pulling the images that are referenced by their digests.
+type RepositoryMirrors struct {
+	// source is the repository that users refer to, e.g. in image pull specifications.
+	// +required
+	Source string `json:"source"`
```

Comment:
which format does this have? Please specify regex, minlength, format, etc.

Reply:
This is not resolved. It needs specification.

Reply:
@QiWang19 Those specifications can be added as comments:

```
// +kubebuilder:validation:Pattern=`^https?:\/\/`
// +kubebuilder:validation:MaxLength=2048
// +kubebuilder:validation:Required
// +required
```

The pattern in this example is just an example... Is it URL format? or host:port format?

Reply:
@rphillips The format is the prefix in the doc  https://github.com/containers/image/blob/main/docs/containers-registries.conf.5.md#choosing-a-registry-toml-table, should I link this doc to the comment

Reply:
the format should be specified like Ryan described above through `kubebuilder:validation:Pattern`